internal/manager/manager.go

```diff
@@ -60,25 +57,49 @@ func Start(cfg config.Config) error {
 		return fmt.Errorf("cannot build runtime manager: %w", err)
 	}
 
-	err = sdk.RegisterGatewayClassController(mgr, gc.NewGatewayClassImplementation(cfg, eventCh))
+	// Register GatewayClass implementation
+	err = sdk.RegisterController[*gatewayv1beta1.GatewayClass](mgr,
```

Comment:
Would it be possible to register multiple implementations in one call, rather than registering each one individually?

Reply:
definitely!